app/code/BVMyBlog/Blog/Api/BlockRepositoryInterface.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
Why BlockRepositoryInterface? Please rename to BlogRepositoryInterface or PostRepositoryInterface. The same is for model